.eslintrc

```diff
@@ -24,7 +24,12 @@
     "no-console": 0,
     "no-alert": 0,
     "object-curly-spacing": 0,
-    "react/jsx-space-before-closing": 0
+    "react/jsx-space-before-closing": 0,
+    "arrow-body-style": 0,
+    "max-len": [1, 120],
+    "no-param-reassign": 0,
+    "consistent-return": 0,
+    "no-use-before-define": 0
```

Comment:
I'd like to leave `consistent-return` and `no-use-before-define` because they move us closer to some static type-checking rules that can avoid some common bugs. I know `no-use-before-define` is annoying with how we're using styles (requiring the `let` statement at the top) but I think the cost in typing may save us down the line...